frontend/src/store/facilityBooking/action.ts

```diff
@@ -263,14 +277,29 @@ export const resetCreateBookingSuccessFailure = (failureBoolean: boolean, succes
   })
 }
 
+/**
+ * Sets is loading
+ * @param desiredState (boolean) TRUE == loading, FALSE == not loading
+ * @returns sets the isloading state according to given state
+ */
 export const setIsLoading = (desiredState: boolean) => (dispatch: Dispatch<ActionTypes>) => {
   dispatch({ type: FACILITY_ACTIONS.SET_IS_LOADING, isLoading: desiredState })
 }
 
+/**
+ * Sets block out
+ * @param desiredState (boolean) TRUE == open, FALSE == close
+ * @returns sets the block out state according to given state
```

Comment:
some code are self-documenting so no need to elaborate much
```suggestion
 * @param desiredState (boolean)
 * @returns sets `blockOutIsOpen` state according to `desiredState`
```